pack/tcu/icc_encounter.json

```diff
@@ -0,0 +1,563 @@
+		"faction_code": "mythos",
+		"flavor": "This nightmare will not end until you stop the one behind it: Anette Mason and the spirit which possesses her. The world is falling apart all around you, and the coven is only bringing the end closer.",
+		"illustrator": "Matthew Cowdery",
+		"name": "Beyond the Grave",
+		"pack_code": "icc",
+		"position": 287,
+		"quantity": 1,
+		"stage": 2,
+		"text": "[free] Remove 3 breaches from this act: Place 1 clue on a random location.\n<b>Objective</b> - If Anette Mason is defeated, advance.",
+		"type_code": "act"
+	},
+	{
+		"back_link": "05288b",
+		"code": "05288a",
+		"clues": 3,
+		"encounter_code": "secrets_of_the_universe",
+		"encounter_position": 1,
+		"faction_code": "mythos",
+		"flavor": "The state of the city continues to decay. Breaches in reality begin to rip through the fabric of the Earth, and a faint melody of discordant pipes can be heard throughout the streets.",
+		"illustrator": "Matthew Cowdery",
+		"name": "Dark Knowledge (v. II)",
+		"pack_code": "icc",
+		"position": 288,
+		"quantity": 1,
+		"stage": 1,
+		"text": "[free] Remove 3 breaches from this act: Place 1 clue on a random location.\n<b>Objective</b> - When the investigators have collected the requisite number of clues, they must immediately spend them and advance. When this act advances, move all of the breaches on it to the next act.",
+		"type_code": "act"
+	},
+	{
+		"code": "05288b",
+		"encounter_code": "secrets_of_the_universe",
+		"encounter_position": 1,
+		"enemy_damage": 1,
+		"enemy_evade": 4,
+		"enemy_fight": 4,
+		"enemy_horror": 3,
+		"faction_code": "mythos",
+		"hidden": true,
+		"illustrator": "Nicholas Elias",
+		"is_unique": true,
+		"health": 6,
+		"health_per_investigator": true,
+		"name": "Carl Sanford",
+		"pack_code": "icc",
+		"position": 288,
+		"quantity": 1,
+		"subname": "Deathless Fanatic",
+		"text": "<b>Spawn</b> - Silver Twilight Lodge.\nHunter. Retaliate.\nCarl Sanford gets -2 health for each clue controlled by an investigator.\n<b>Forced</b> - After any number of breaches are removed from the current act: Place 1 of them on Carl Sanford's location.",
+		"traits": "Humanoid. Silver Twilight. Elite.",
+		"type_code": "enemy",
+		"victory": 2
+	},
+	{
+		"back_flavor": "Using force and ingenuity, you have managed to defeat Carl Sanford at his own game. The power he was drawing from surges painfully through him. \"You imbeciles,\" he spits, \"what do you think you have accomplished? We could have been immortal. Don't you understand? We could have been gods!\" he struggles to remain on his feet, his strength waning as he loses control of his power.\n\"At what cost?\" You reply, taking in the sight before you. \"You would have made us slaves, not gods. Slaves to your greed and the Order's power. I'd rather stay mortal than be your thrall.\"\n\"Then we will all perish,\" Carl responds bitterly. \"I could have made us into celestials. Instead we will be extinct. I hope you are satisfied.\"",
```

Comment:
Please add underline tags for 'gods' - it's emphasised in the flavor text, but I reckon an underline will do!